source/clear-linux/tutorials/wp-install/wp-install.rst

```diff
@@ -3,204 +3,150 @@
 Create a Clear Linux WordPress\* server
 #######################################
 
-Prerequisites
-=============
+This tutorial guides you through installing WordPress components on your |CL| LAMP server. During :ref:`Creating a Clear Linux based web server <web-server-install>`, you created the initial WordPress MySQL database `WordPress` using phpMyAdmin\* and MariaDB\*. Now that the LAMP server is up and running, you can add the WordPress components needed to host a WordPress website on your host system.
 
-This tutorial assumes you have already
-:ref:`installed Clear Linux on a bare metal system <bare-metal-install>`
-and that you have :ref:`set up a LAMP web server <web-server-install>`.
+This tutorial assumes you have successfully installed :ref:`Clear Linux on bare metal<bare-metal-install>` and that you have :ref:`set up a LAMP web server <web-server-install>`.
 
-Additionally, all the steps on this tutorial were tested using a NUC6i5SYH
-Intel® NUC. Visit the `NUC6i5SYH product page`_ for detailed information.
+Before you install new packages, update the |CL| OS with the following command:
 
-Before installing any new packages, update the |CL| OS with the
-console command:
-
-.. code-block:: console
+.. code-block:: bash
 
    sudo swupd update
 
 
-Creating a WordPress server
+Create a WordPress server
 ===========================
 
-This tutorial walks you through the process of installing the WordPress
-components on your |CL| LAMP server. At the end of the
-:ref:`web server tutorial <web-server-install>`, we created the initial
-WordPress MySQL database `WordPress` using phpMyAdmin\* and MariaDB\*.
-With the LAMP server up and running, we can add the WordPress components
-needed to host a WordPress website on your host system.
-
-Throughout this tutorial we will reference your website name as
-<your_website>. There are numerous articles available on-line regarding how
-to name your website and acquire the necessary certificates. Those tasks are
-beyond the scope of this tutorial.
-
-There are also several actions you can take to harden your website from
-attacks. The security of your website and the data it contains are complex
-and ever-evolving tasks. They should be at the top of your to do list if you
-plan to expose your website to the outside world. This tutorial does not
-address security measures that you can take to harden your site but we
-strongly encourage you to look into it.
-
-Installing WordPress
---------------------
+WordPress can be installed in a variety of ways. These instructions are written for users who have installed phpMyAdmin.
```

Comment:
Let's consider a different approach. Since your revision recommends moving the original first paragraph of this section to the top of the page, there's no need to repeat our user expectations again. Instead, let's salvage the remaining two paragraphs. There's good stuff in there--valuable disclaimers and guidance for beginners. For example, consider this revision:

"Before deploying a website or blog, it's best to organize and plan carefully. For example, you may need to register a domain name and acquire certificates depending on who provides hosting, review and harden security, perform pilot testing, and engage in several other activities. However, all of these activities are beyond the scope of this tutorial." 

You could use this or write your own version.

Reply:
I see what you mean. I revised this section. Thanks for your input.